payment-service/src/main/resources/application.yaml

```
@@ -0,0 +1,15 @@
+stripe:
+  public: pk_test_51OZH3GCaEc3yJYRQrpROT6ZBZFXXMAfuOfEFy6MuHO0L0IDOtLuwSDEPzsbJotQSiqZb1wLZWVsYDP70DmYty3dE00MEUPES9e
+  secret: sk_test_51OZH3GCaEc3yJYRQFoi2ZeTXoGciOObsP3quiPkMqA7BunEaD1GB9Nva8wh31m9lBKPv4w9fD0VNRn6eKGZ9ZFZJ00Yn21tWIv
```

public и secret key - сенсетив информация
```suggestion
  stripe:
  public: ${PUBLIC_KEY}
  secret: ${SECRET_KEY}
```